content/posts/fleek-network-how-to-get-the-latest-updates-for-ursa-cli-from-the-source-repository.md

```diff
@@ -0,0 +1,291 @@
+---
+template: post
+draft: false
+title: 'Fleek Network: How to get the latest updates for Ursa CLI from the source repository'
+slug: fleek-network-how-to-get-the-latest-updates-for-ursa-cli-from-the-source-repository
+date: 2022-12-09T13:00:00.000+00:00
+socialImage: https://storageapi.fleek.co/fleek-team-bucket/fleek-network-how-update-ursa-cli-from-git-repo.png?202212091140
+canonical: ''
+description: How to get the latest updates for Ursa CLI from the source repository
+category: Tutorial
+tags:
+- CDN
+- Guide
+- Getting Started
+- Fleek Network
+- GIT
+
+---
+
+![](https://storageapi.fleek.co/fleek-team-bucket/fleek-network-how-update-ursa-cli-from-git-repo.png?202212091140)
+
+
+## Introduction
+
+Fleek Network's Ursa CLI soure-code 🤖 is updated frequently and thus keeping up with changes can be a bit of a chore and specially difficult for users who are trying to have it compiled for the first time, or updating the Ursa CLI binary in their operating systems. While we should have stable releases in the future at the current phase of development, there's a requirement to follow the contributions directly in the repository: checking in and out, the commits you're interested in 🧐.
+
+In this guide, we’ll have a simple look into how to pull changes and update the Ursa CLI binary for Linux, macOS and Windows (WSL): for the linux guides we’ll use Ubuntu as an example. 
+
+## Prerequesites
+
+To follow this guide, you will need:
+
+- Some experience running the command-line interface
+- Have Git installed correctly
+- Bit of experience with Git version control
+- Docker (if you're following that path)
+
+We're also assuming that you have followed our initial [getting started guide](https://blog.fleek.co/posts/fleek-network-getting-started-guide). If you haven't already 😅, read it [here](https://blog.fleek.co/posts/fleek-network-getting-started-guide).
+
+### Table of Contents
+
+- [Pulling the latest changes](#pulling-the-latest-changes)
+- [How to rebuild Ursa CLI?](#how-to-rebuild-ursa-cli)
+  - [On host notes](#on-host-notes)
+  - [Docker notes](#docker-notes)
```

Comment:
Maybe:
- `Native build`
- `Docker build`
Would be better section titles

Reply:
Sure, sounds alright!